src/index.coffee

```diff
@@ -1,17 +1,24 @@
+escape = require('js-string-escape');
+
 module.exports = class AngularTemplatesCompiler
   brunchPlugin: yes
   type: 'template'
   extension: 'html'
 
+  _default_path_transform: (path) ->
+    # Default path transformation is a no-op
+    path
+
   constructor: (config) ->
     @module = config.plugins?.angular_templates?.module or 'templates'
+    @path_transform = config.plugins?.angular_templates?.path_transform or @_default_path_transform
 
   parseHtml: (str) ->
```

Comment:
Which part? I think it does need to check whether there's a path_transform set in the config and default to using the built-in no-op. Could move the check down to the place it's used but seems better to keep all the config variables together?